pkg/proxy/ipvs/proxier.go

```diff
@@ -497,6 +487,8 @@ func CanUseIPVSProxier(handle KernelHandler, ipsetver IPSetVersioner) (bool, err
 	}
 	wantModules := sets.NewString()
 	loadModules := sets.NewString()
+	linuxKernelHandler := NewLinuxKernelHandler()
+	_, ipvsModules, _ := utilipvs.GetKernelVersionAndIPVSMods(linuxKernelHandler.executor)
```

Comment:
Check err ?

Reply:
Interestingly upstream chose to explicitly ignore errors.

Basically we're only interested in the 2nd return value, and in error paths it is nil, so ipvsModules will be nil. This leads to `wantModules` being empty, and then potentially the `len(modules)` bit fails lower down. So there's something handling it it seems. Haven't looked at the details of `mods, err := handle.GetModules()`, however I not particularly tempted to deviate from upstream just for this.

Reply:
ok